### DISCUSSION TITLE
Should we rearrange all dialect folders into a folder called "databases"?

### DISCUSSION THREAD

**Discussion by thadguidry** [9tigerio/db2rest#730] [Ideas]
## Pros
- easy to remember top-level folder "databases" while retaining the "xxxx-dialect" subfolders under it.
- shorter folder view hierarchy making things looks easier for new contributors
- smaller hierarchy let's README showup a bit better on GitHub repo
- faster navigation in IDE since most changes are not done within dialects verbosing, but piecemeal.

## Cons
- small effort?

**Comment by kdhrubo** [comment]
Yes we can fold them into 1 top-level folder called dialect with packages for each supported database
Should not be simple and not time consuming, have to see what impact( if any ). Will it pull unnecessary dependencies like mongo or MySQL if someone uses it just as a library later. It will also speed up build time.
